apps/C/graphic_screen_test.c

```diff
@@ -1,14 +1,13 @@
+#include "../defines/rvc_defines.h"
```

Comment:
Why everything is still defined in the file? 
Dont you use the "#include "../defines/rvc_defines.h""

Reply:
Taken care of.